src/core/dbus.c

```diff
@@ -979,6 +989,19 @@ int bus_init_private(Manager *m) {
         return 0;
 }
 
+static int bus_track_stash(sd_bus_track *t, char ***l) {
+        for (const char *n = sd_bus_track_first(t); n; n = sd_bus_track_next(t)) {
+                int c;
+
+                c = sd_bus_track_count_name(t, n);
+                for (int j = 0; j < c; j++)
+                        if (strv_extend(l, n) < 0)
+                                log_oom();
+        }
+
+        return 0;
+}
```

Comment:
or alternatively, this seems somewhat generic, hence maybe move this into src/shared/bus-util.[ch] and then do keep the sd_bus_track_count_name() stuff in place. call it bus_track_to_strv() or so?